pipeline-modules/deployment-service/aws/basic-eks-regional.yaml

```diff
@@ -198,6 +198,10 @@ template: |
           {% endif %}
           rm -f values.yaml;
 
+          if [ -z "$docker_registry" ]; then
+          export docker_registry="index.docker.io"
```

Comment:
isn't this supposed to be a url?

Reply:
actually maybe it is just the host - the code is confusing because we seem to have this function "GetRegistryUrl" that isn't called anywhere but the GetHost() call returns url.URL....